### PR TITLE
Bump version to v2.14.0

### PR DESCRIPTION
**Pull request by hhaAndroid**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+- Add `simple_test` to dense heads to improve the consistency of the inference interface of single-stage and two-stage
+- Revert the `test_mixins` to test image by image to improve efficiency and readability
+- Add Faster R-CNN and Mask R-CNN 3x mstrain config
+- Stable support of  exporting DETR to ONNX with dynamic shapes and batch inference
+
+## New Features
+
+- Support pretrained models from MoCov2 (#5286)
+- Add Faster R-CNN and Mask R-CNN 3x mstrain config (#5179,  #5233)
+- Stable support of  exporting DETR to ONNX with dynamic shapes and batch inference (#5168)
```
Comment (RangiLyu) [inline docs/changelog.md:16]: space between `of` and `exporting`

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+- Make YOLOv3 Neck more flexible (#5218)
+- Refactor SSD to make it more general (#5291)
+- Refactor `anchor_generator` and `point_generator` (#5349)
+- Allow to configure out the `mask_head` of the HTC algorithm (#5389)
+- Delete deprecated warning in FPN (#5311)
+- Move `model.pretrained` to `model.backbone.init_cfg` (#5370)
+- Make deployment tools more friendly to use (#5280)
+- Clarify installation documentation (#5316)
+- Add ImageNet Pretrained Models docs (#5268)
+- Add FAQ about  training loss=nan solution and COCO AP or AR =-1 (# 5312, #5313)
```
Comment (RangiLyu) [inline docs/changelog.md:41]: space between `about` and `training`

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
 ## Changelog
 
+# v2.14.0 (29/6/2021)
+
+## Highlights
+
+- Add `simple_test` to dense heads to improve the consistency of the inference interface of single-stage and two-stage
```
Comment (ZwwWayne) [inline docs/changelog.md:7]: of single-stage and two-stage detectors.

File: docs/changelog.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+- Revert the `test_mixins` to test image by image to improve efficiency and readability
```
Comment (ZwwWayne) [inline docs/changelog.md:8]: process image by image in the head?

File: docs/changelog.md [hunk 5]
```diff
@@ -1,5 +1,45 @@
...
+- Add Faster R-CNN and Mask R-CNN 3x mstrain config
```
Comment (ZwwWayne) [inline docs/changelog.md:9]: config using multi-scale training with 3x schedule

File: docs/changelog.md [hunk 6]
```diff
@@ -1,5 +1,45 @@
...
+- Stable support of exporting DETR to ONNX with dynamic shapes and batch inference
+
+## New Features
+
+- Support pretrained models from MoCov2 (#5286)
```
Comment (ZwwWayne) [inline docs/changelog.md:14]: from MoCo v2 and SwAV

File: docs/changelog.md [hunk 7]
```diff
@@ -1,5 +1,45 @@
...
+- Add Faster R-CNN and Mask R-CNN 3x mstrain config (#5179, #5233)
```
Comment (ZwwWayne) [inline docs/changelog.md:15]: see comment above.

File: docs/changelog.md [hunk 8]
```diff
@@ -1,5 +1,45 @@
...
+- Stable support of exporting DETR to ONNX with dynamic shapes and batch inference (#5168)
+
+## Bug Fixes
+
+- Fix size mismatch bug in `multiclass_nms` (#4980)
+- Fix the import path of `MultiScaleDeformableAttention` (#5338)
+- Fix GCNet X101 model config error (#5360)
```
Comment (ZwwWayne) [inline docs/changelog.md:22]: Fix errors in config of GCNet ResNext101 models

File: docs/changelog.md [hunk 9]
```diff
@@ -1,5 +1,45 @@
...
+- Fix Grid-RCNN error when there is no bbox result (#5357)
+- Fix web links ending with `.md` (#5315)
+
+## Improvements
+
+- Add `simple_test` to dense heads to improve the consistency of the inference interface of single-stage and two-stage (#5264)
```
Comment (ZwwWayne) [inline docs/changelog.md:28]: See comment above.

File: docs/changelog.md [hunk 10]
```diff
@@ -1,5 +1,45 @@
 ## Changelog
 
+# v2.14.0 (29/6/2021)
+
+## Highlights
+
+- Add `simple_test` to dense heads to improve the consistency of the inference interface of single-stage and two-stage
+- Revert the `test_mixins` to test image by image to improve efficiency and readability
+- Add Faster R-CNN and Mask R-CNN 3x mstrain config
+- Stable support of exporting DETR to ONNX with dynamic shapes and batch inference
```
Comment (ZwwWayne) [inline docs/changelog.md:10]: We do not need to highlight this.

File: docs/get_started.md [hunk 11]
```diff
@@ -12,6 +12,7 @@ The compatible MMDetection and MMCV versions are as below. Please install the co
 | MMDetection version |    MMCV version     |
 |:-------------------:|:-------------------:|
 | master              | mmcv-full>=1.3.3, <1.4.0 |
+| 2.14.0              | mmcv-full>=1.3.3, <1.4.0 |
```
Comment (ZwwWayne) [inline docs/get_started.md:15]: Should use 1.3.8 because we fix issues of the multiscale transformer.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**